docs/permissions/plugin-authors/01-setup.md

```diff
@@ -37,8 +40,8 @@ The source code is available here:
 2.  Add these packages as dependencies for your Backstage app:
 
     ```
-    $ yarn workspace backend add @internal/plugin-todo-list-backend@^1.0.0 @internal/plugin-todo-list-common@^1.0.0
-    $ yarn workspace app add @internal/plugin-todo-list@^1.0.0
+    $ yarn workspace backend add @internal/plugin-todo-list-backend @internal/plugin-todo-list-common
```

Comment:
These commands still fail for me. Adding the dependencies manually to package.json works though!